src/System.Device.Gpio/System/Device/Gpio/Drivers/UnixDriver.cs

```diff
@@ -4,8 +4,15 @@
 
 namespace System.Device.Gpio.Drivers
 {
+    /// <summary>
+    /// The base class for the standard unix drivers
+    /// </summary>
     public abstract class UnixDriver : GpioDriver
     {
+        /// <summary>
+        /// Construct an instance of an unix driver.
+        /// Just checks that the platform is Unix.
```

Comment:
NIT: I would probably ommit the fact that all we do here is check platform as we may change this in the future and will likely miss an update to the summary